jenkins/runners/systemd-pr-build-vagrant-sanitizers.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+function at_exit() {
+    # Correctly collect artifacts from all sanitizer jobs and generate a nice
+    # directory structure
+    mkdir _artifacts_all
+    mv artifacts_* _artifacts_all
+    mv _artifacts_all artifacts_all
+
+    utils/generate-index.sh artifacts_all index.html
+}
+
+trap at_exit EXIT
+
+export PATH="/home/systemd/bin:$PATH"
+ARGS=
+
+if [ "$ghprbPullId" ]; then
+    ARGS="$ARGS --pr $ghprbPullId "
+fi
+
+git clone https://github.com/systemd/systemd-centos-ci
+cd systemd-centos-ci
```

Comment:
I guess this makes things slower... but transparency is more important.

Reply:
This was, unfortunately, even in the original inline script, because the workspace is the systemd repo *context*, when triggered via systemd PR (i.e. when a job is triggered via PR event from the systemd repository, the root of the workspace is actually root of the systemd repository checked out to a given PR).